Exercise_1.py

```diff
@@ -2,19 +2,23 @@ class myStack:
   #Please read sample.java file before starting.
   #Kindly include Time and Space complexity at top of each file
      def __init__(self):
+       self.l = list()
          
      def isEmpty(self):
-         
+         return len(self.l) == 0
      def push(self, item):
+       self.l.append(item)
```

Comment:
What if stack overflows?

Reply:
if len(l) > maxsize:
     return -1

will check for this return value in main and will print("stack is at its capacity, cannot insert").